vaccine_feed_ingest/runners/ok/vaccinate_gov/parse.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+# adapted from ca/sf_gov/parse
+
+import json
+import pathlib
+import sys
+
+output_dir = pathlib.Path(sys.argv[1])
+input_dir = pathlib.Path(sys.argv[2])
+
+json_filepaths = input_dir.glob("*.json")
+
+for in_filepath in json_filepaths:
```

Comment:
I think you can replace this whole file with the [built-in infra for ndjson](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/JSON-to-NDJSON-Parser) since it doesn't require JSON traversal (unlike the CA file). Something like:

```
---
state: ok
site: vaccinate_gov
parser: json_list
```

Reply:
thanks! fixed :)